cypress/integration/mocked/verify_email.spec.ts

```diff
@@ -284,25 +284,18 @@ describe('Verify email flow', () => {
       cy.contains(VerifyEmail.CONTENT.CONFIRM_EMAIL);
       cy.contains(verifiedUserWithNoConsent.user.primaryEmailAddress);
 
-      cy.intercept({
-        method: 'POST',
-        url: 'https://www.google.com/recaptcha/api2/**',
-        times: 1,
+      cy.intercept('POST', 'https://www.google.com/recaptcha/api2/**', {
+        statusCode: 500,
       });
 
-      // click on send link button and expect reCAPTCHA errors for first two clicks.
       cy.contains(VerifyEmail.CONTENT.SEND_LINK).click();
       cy.contains('Google reCAPTCHA verification failed. Please try again.');
 
+      // show extended reCaptcha message on the second failure
       cy.contains(VerifyEmail.CONTENT.SEND_LINK).click();
       cy.contains('Google reCAPTCHA verification failed.');
       cy.contains('If the problem persists please try the following:');
       cy.contains('userhelp@');
-
-      cy.contains(VerifyEmail.CONTENT.SEND_LINK).click();
```

Comment:
I think this test was checking to see if we can still put the form through when reCAPTCHA eventually succeeds, by removing this we're no longer checking that. Maybe we could intercept it and show it as a successful check after the second message is shown so we can this final assertion in there too?

Reply:
This would reintroduce the flakiness as there is no way to know in advance if reCaptcha will make 2 or 3 calls to the API

Reply:
aha, I see. probably not a big deal as we check the form submission with recaptcha in other places 👍